Trees/FenwickTreeImplementation.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: Mohit Kumar
+ * Fedwick Tree Implementation in JavaScript
+ *  Fedwick Tree Implementation for finding prefix sum.
+*/
+
+function constructFenwickTree(array, n) {
+  for (let i = 1; i <= n; i++) {
+    FeneickTree[i] = 0;
+  }
+  for (let i = 0; i < n; i++) {
+    updateFenwickTree(n, i, array[i]);
+  }
+}
+
+function updateFenwickTree(n, index, value) {
+  index = index + 1;
+  while (index <= n) {
+    FeneickTree[index] += value;
+    index += index & (-index);
+  }
+}
+ 
+function getPrefixSum(index) {
+  let currSum = 0;
+  index = index + 1;
+  while (index > 0) {
+    currSum += FeneickTree[index];
+    index -= index & (-index);
+  }
+    
+  return currSum;
+}
+
+let MAX_VALUE = 1000;
+let FeneickTree = new Array(MAX_VALUE);
+let array = [3, 2, 0, 6, 5, -1, 2];
+
+let length = array.length;
+constructFenwickTree(array, length);
+
+getPrefixSum(6);
```

Comment:
Testing is done in a separate `jest` file

Reply:
1. Added file FenwickTreeImplementation.test.js for testing purposes where the functionality of Fenwick Tree is tested.
2. Removed all the global variables from FenwickTreeImplementation.js 
3. Formatted all the code with standard.js
4. Tested code using jest